Bookish/Bookish.Web/Models/AddBookViewModel.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Bookish.Web.Models
+{
+    public class AddBookViewModel
+    {
+        public string Isbn { get; }
+        public string Title { get; }
+        public string Authors { get; }
+        public int Copies { get; }
+        public IEnumerable<string> ErrorMessage { get; private set; }
```

Comment:
Does this need a private set? Can't it just have a get like the other properties? 🙂